week_1/week_1.txt

```diff
@@ -0,0 +1,7 @@
+2 APRIL
+    HTML Tags
+    HTML Structure 
+    Chrome Dev Tools 
+    CSS Basics
```

Comment:
Same here and for others please add what all things you have learn in detail like 
- css basic
     - display
     - font-size
     - ...
     - ...
....